sdk/typescript/runtime/runtime_bun.go

```diff
@@ -88,6 +88,8 @@ func (b *BunRuntime) SetupContainer(ctx context.Context) (*dagger.Container, err
 	// Merge all the generated files together and setup an entrypoint command.
 	return runtimeWithDep.ctr.
 		WithMountedDirectory(GenDir, sdkLibrary).
+		// Make @dagger.io/dagger resolvable for ts-introspector (it doesn't read tsconfig paths).
```

Comment:
That's curious we need that but since that the same client, I guess it works.

I'm wondering (for a follow up) if we could remove the `sdk` dir locally and just put the bundle version in the `node_modules` so we don't duplicate that logic here.

Normally your addition shouldn't impact performances since its mounted.